packages/mobile/src/screens/sign-on-screen/screens/SelectArtistScreen/SelectArtistsScreen.tsx

```diff
@@ -58,13 +58,10 @@ export const SelectArtistsScreen = () => {
   )
 
   const handleSubmit = useCallback(() => {
+    // This call is what eventually triggers the RootScreen to redirect to the home page (via conditional rendering)
+    dispatch(finishSignUp())
```

Comment:
was getting stuck in the infinite load loop when I realized this action needs to be dispatched no matter the loading state

Reply:
Nice call